projects/npm-tools/packages/npm-scripts/src/scripts/createBridges.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const {addNamespace} = require('@liferay/js-toolkit-core');
+const fs = require('fs');
+const path = require('path');
+
+module.exports = function (bridges, dir) {
+	const projectPackageJson = JSON.parse(
+		fs.readFileSync('package.json', 'utf-8')
+	);
+
+	Object.entries(bridges).forEach(([packageName, exportName]) => {
```

Comment:
Don't forget that you rarely need `forEach()` nowadays:

```js
for (const [packageName, exportName] of Object.entries(bridge)) {
```

Avoids a little bit of function call overhead, and is more flexible in general (eg. permits early `return`).